src/site/content/en/blog/introducing-libsquoosh/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introducing libSquoosh
+subhead: It's Squoosh, but as a Node library
+description: |
+  It's Squoosh, but as a Node library
+date: 2021-06-02
+authors:
+  - surma
+hero: image/i9nJGvw3SnTPH63zKOYWtI6cP5m2/zTPjhYuOfh8bnk7c7tlO.png
+thumbnail: image/i9nJGvw3SnTPH63zKOYWtI6cP5m2/zTPjhYuOfh8bnk7c7tlO.png
+alt: $ npm install --save @squoosh/lib
+tags:
+  - blog
+  - tools
+  - javascript
+  - images
+---
+
+**We are happy to introduce [libSquoosh], an experimental Node library on top of which the Squoosh CLI is built, giving you all the capabilities for the Squoosh CLI with a JavaScript-idiomatic interface.**
+
+Squoosh.app is a PWA that compresses images for you in the browser. It supports many old and new image formats and processes them client-side in the browser through WebAssembly. This means your pictures stay safely on your own computer rather than being sent to a server somewhere, and means that Squoosh works even when offline.
```

Comment:
```suggestion
Squoosh.app is a PWA that compresses images for you in the browser. It supports many old and new image formats and processes them client-side in the browser through WebAssembly. This means your pictures stay safely on your own computer rather than being sent to a server somewhere, and that Squoosh works even when offline.
```